app/controllers/application_controller.rb

```diff
@@ -15,4 +16,26 @@ def sign_in_user
   def current_user
     @current_user ||= sign_in_user&.user
   end
+
+  def support_user
+    @support_user ||= sign_in_user&.support_user
+  end
```

Comment:
Do we need a separate `support_user` method here? It feels a bit weird that the controller will have both a `current_user` and a `support_user` which will represent the same thing in the case of a support user being logged in.

I feel like it'd probably be sufficient to use the existing `current_user` object and authorise access with:

```ruby
if current_user.support_user?
  # user is allowed access to support things
else
  # 🙅🏻‍♂️
end
```

And, as a bit of an aside – not necessarily something to change in this PR:

I'm now wondering if we really need the `class Placements::SupportUser < Placements::User` at all. 🤔 It's effectively just a scope. A `Placements::SupportUser` is functionally the same as a `Placements::User` at this point.

This may change in a future refactor if we decide that the organisations list should be generated by the model as `user.organisations` or `support_user.organisations` because that behaviour would be different.

Reply:
I suppose I was assuming there would be methods on the `support_user` model that would be distinct from the `current_user` (eg, like the organisations you mention). I'm happy to go either way.

Reply:
I think this might give us a more elegant way forward, where we don't need to define a secondary `support_user` method.

PR: https://github.com/DFE-Digital/itt-mentor-services/pull/45